.circleci/config.yml

```diff
@@ -73,6 +73,52 @@ jobs:
         - "d3:8f:a8:6e:b6:ef:37:65:1a:dc:2b:88:3b:ff:50:f4"
     - run: bin/publish-coverage
 
+  # Copy-pasta from dd-trace-java. Thank you <3
+  system-tests:
+    parameters:
+      systemTestsCommit:
+        type: string
+        default: main
+    machine:
+      image: ubuntu-2004:current
```

Comment:
It may be interesting to update our ci image and use a `docker` executor instead... but it's docker inside docker, it might be problematic.